Sources/MintKit/MintKit.swift

```diff
@@ -106,7 +106,20 @@ public struct Mint {
         }
         try package.path.mkpath()
 
-        if !force && package.commandPath.exists && !package.version.isEmpty {
+        if package.version.isEmpty {
+             // we don't have a specific version, let's get the latest tag
+            let tags = try shellOut(to: "git ls-remote --tags \(package.git)")
+            if tags.isEmpty {
+                package.version = "master"
+                print("🌱  Using \(package.name) \(package.version.quoted)")
+            } else {
+                let latestTag = String(tags.split(separator: "\n").last!.split(separator: "\t").last!.split(separator: "/").last!)
```

Comment:
@yonaskolb,

The tags are sorted alphabetically. The last one may not be the lastest. You will have to parse the versions and sort them yourself.

For example, right now 0.9.2 ends up after 0.24.0 for SwiftLint:

```shell
$ git ls-remote --tags https://github.com/realm/SwiftLint
...
224c768443f737dee79f80e23d360f57475ffcff	refs/tags/0.1.1
76939117e8db4cff7ccd5d362340cbc6a7af878c	refs/tags/0.1.1^{}
d591ff27dcbc759384cbb20cc7a200b9cabe7b7c	refs/tags/0.1.2
4c91500e7b90a1d651d16e9c604ab927da050e8f	refs/tags/0.1.2^{}
acf15ad09f33362bf6e7ff9ffbcc6f725c6db122	refs/tags/0.10.0
fbd2bcd0edc264e04d02f87b0d0f98add754ac88	refs/tags/0.10.0^{}
9fa96ecd20bf0ec874969ed055845a8ec1f4c257	refs/tags/0.11.0
3ec8fee271fc509d17989964c595524a8ba04599	refs/tags/0.11.0^{}
...
b671ab3f77e5cb99b4bb197dd765dc995bd0b311	refs/tags/0.23.1
c34c5c164954132a87e90d2312a74826a54487ed	refs/tags/0.23.1^{}
8d4790f4c1ff272abbac342202fed64e450b44f2	refs/tags/0.24.0
06ece1ea8dbeb3bc421e54f907e906cac32038d5	refs/tags/0.24.0^{}
ff4a0cb285609c0330f94fc14040c65ae0e981a4	refs/tags/0.3.0
ae595d67edc24383446a346e218d66a00994bab5	refs/tags/0.3.0^{}
490d26e85d9410754e39207b276139f7560da481	refs/tags/0.4.0
1201a9e38cddb842aee7ad8ae48078e1d722f39f	refs/tags/0.4.0^{}
...
6be1abf923cec24ab04ae1a1101dc47112263012	refs/tags/0.9.1
0d6ed5c71e7eafdb5f7a7cdecf9f3ae09e404ee3	refs/tags/0.9.1^{}
dbe700666547ec3474e83155546b08151dc7686c	refs/tags/0.9.2
6cafa08e70dab64c39c38ea5b9542126389d27aa	refs/tags/0.9.2^{}
```